### DISCUSSION TITLE
Sony headphones WF-C500 are not visible on Bluetooth

### DISCUSSION THREAD

**Discussion by JoaoLucasMoraisOrtiz** [elementary/triage#330] [Q&A]
I just switched from Fedora to elementaryOS. Previously, on Fedora, I used these same headphones with my laptop, but now they aren't visible even though the system has been searching for new devices for several minutes. I tried following the AirPods tip, but it didn't work for my headphones. Any help?

**Comment by RogerWeihrauch** [comment]
hi
hm, from given information I can only think of:
1) are you sure that latest BT drivers/modules are installed definitely? (like: 'apt search/install bluez/bluetooth..')
2) did pairing work successfully/were you able to start pairing?
3) are you sure bt was hardware-enabled on your device (and not switched off by key combination/pressed)?
regards
Roger

**Reply by JoaoLucasMoraisOrtiz** [reply]
Hello, thanks for your answer.
My problem wasn't with the OS but with my headphones. With this model, you need to press the button for 5 seconds to start pairing; they don't do it automatically.
Thanks for your answer and attention.
